01-messaging.md

```diff
@@ -22,49 +22,49 @@ All data fields are big-endian unless otherwise specified.
+   the message and that conforms to a format matching the `type`.
 
 The `type` field indicates how to interpret the `payload` field.
-The format for each individual type is specified in a specification in this repository.
+The format for each individual type is defined by a specification in this repository.
 The type follows the _it's ok to be odd_ rule, so nodes MAY send _odd_-numbered types without ascertaining that the recipient understands it.
 
 A sending node:
-  - MUST NOT send an evenly-typed message not listed here, without prior negotiation.
+  - MUST NOT send an evenly-typed message not listed here without prior negotiation.
 
 A receiving node:
   - upon receiving a message of _odd_, unknown type:
     - MUST ignore the received message.
   - upon receiving a message of _even_, unknown type:
     - MUST fail the channels.
 
-The messages are grouped logically into four groups, ordered by their most significant set bit:
+The messages are grouped logically into four groups, ordered by the most significant bit that is set:
 
   - Setup & Control (types `0`-`31`): messages related to connection setup, control, supported features, and error reporting. These are described below.
   - Channel (types `32`-`127`): messages used to setup and tear down micropayment channels. These are described in [BOLT #2](02-peer-protocol.md).
-  - Commitment (types `128`-`255`): messages related to updating the current commitment transaction, which includes adding, revoking, and settling HTLCs, as well as updating fees and exchanging signatures. These are described in [BOLT #2](02-peer-protocol.md).
-  - Routing (types `256`-`511`): node and channel announcements, as well as any active route exploration. These are described in [BOLT #7](07-routing-gossip.md).
+  - Commitment (types `128`-`255`): messages related to updating the current commitment transaction, which includes adding, revoking, and settling HTLCs as well as updating fees and exchanging signatures. These are described in [BOLT #2](02-peer-protocol.md).
+  - Routing (types `256`-`511`): messages containing node and channel announcements, as well as any active route exploration. These are described in [BOLT #7](07-routing-gossip.md).
 
-The size of the message is required to fit into a 2-byte unsigned int by the transport layer; therefore, the maximum possible size is 65535 bytes.
+The size of the message is required by the transport layer to fit into a 2-byte unsigned int; therefore, the maximum possible size is 65535 bytes.
 
 A node:
-  - MUST ignore any additional data within a message, beyond the length it expects for that type.
+  - MUST ignore any additional data within a message beyond the length that it expects for that type.
   - upon receiving a known message with insufficient length for the contents:
     - MUST fail the channels.
-  - that understands an option (each is numbered):
+  - that understands a numbered option:
     - MUST include all the fields annotated with that option.
```

Comment:
Hmm, let's just remove "(each is numbered)", since they aren't any more.  I'll typo fix this separately.